docs/data/migration/migration-pickers-v7/migration-pickers-v7.md

```diff
@@ -474,3 +474,12 @@ If you have customized those translation keys, you have to update them following
 +  value == null ? null : value.format('hh:mm:ss')
 +);
 ```
+
+## `FieldValueType` renaming
```

Comment:
We can probably move it to [this section](https://github.com/mui/mui-x/blob/master/docs/data/migration/migration-pickers-v7/migration-pickers-v7.md#renamed-variables).